libs/stdlib/iec61131-st/to_bit.st

```diff
@@ -0,0 +1,246 @@
+FUNCTION TO_BOOL<T: ANY> : BOOL
+    VAR_INPUT
+        in : T;
+    END_VAR
+END_FUNCTION
+
+FUNCTION TO_BOOL__BYTE : BOOL
+    VAR_INPUT
+        in : BYTE;
+    END_VAR
+
+    TO_BOOL__BYTE := BYTE_TO_BOOL(in);
+END_FUNCTION
+
+FUNCTION TO_BOOL__WORD : BOOL
+    VAR_INPUT
+        in : WORD;
+    END_VAR
+
+    TO_BOOL__WORD := WORD_TO_BOOL(in);
+END_FUNCTION
+
+FUNCTION TO_BOOL__DWORD : BOOL
+    VAR_INPUT
+        in : DWORD;
+    END_VAR
+
+    TO_BOOL__DWORD := DWORD_TO_BOOL(in);
+END_FUNCTION
+
+FUNCTION TO_BOOL__LWORD : BOOL
+    VAR_INPUT
+        in : LWORD;
+    END_VAR
+
+    TO_BOOL__LWORD := LWORD_TO_BOOL(in);
+END_FUNCTION
+
+// ======================================================================================================== //
+
+FUNCTION TO_BYTE<T: ANY> : BYTE
+    VAR_INPUT
+        in : T;
+    END_VAR
+END_FUNCTION
+
+FUNCTION TO_BYTE__BOOL : BYTE
+    VAR_INPUT
+        in : BOOL;
+    END_VAR
+
+    TO_BYTE__BOOL := BOOL_TO_BYTE(in);
+END_FUNCTION
+
+FUNCTION TO_BYTE__CHAR : BYTE
+    VAR_INPUT
+        in : CHAR;
+    END_VAR
+
+    TO_BYTE__CHAR := CHAR_TO_BYTE(in);
+END_FUNCTION
```

Comment:
We could probably also add `TO_BYTE__WCHAR` here and have it behave the same as `TO_DWORD__LWORD`. Might be nice if one is dealing with UART.